dynamiqs/utils/operators.py

```diff
@@ -104,7 +104,8 @@ def eye_like(
         dims _(tuple of ints or None)_: Dimensions of each subsystem in the composite
             system Hilbert space tensor product. Defaults to `None` (`x.dims` if
             available, single Hilbert space `dims=(n,)` otherwise).
-        layout _(dq.dense, dq.dia or None)_: Matrix layout.
+        layout _(dq.dense, dq.dia or None)_: Matrix layout. By default, the layout is
+            the same is `x`
```

Comment:
I quite like the phrasing of [`np.zeros_like`](https://numpy.org/doc/2.2/reference/generated/numpy.zeros_like.html) here, something like the following maybe?
```
Overrides the matrix layout of the result. If `None`, the layout is the same as `x`.
```